### PR TITLE
Fine tune BERT model on the full parquet dataset.

### PR DESCRIPTION
**Pull request by simpleParadox**
The BERT pretrained model is has now been finetuned on the full dataset, along with code to calculate the top-k similar records for a given seed record.

The following files are of utmost importance.
1. src/models/transformers-fine-tune.py
2. src/models/custom_model.py
3. There's also a file called app.py inside src/demos that hosts a demo app based on gradio. This isn't paramount to the functioning of the application but is a quick way to understand the capabilities of the model.

The BERT model that is being currently used is bert-base-uncased.

#1

### CODE REVIEW

**Review by xinli-cai — APPROVED** [review]
The modifications look promising. I'd recommend uploading the fine-tuned BERT word embeddings to AWS S3 to align with existing solutions.
